pulp_python/app/models.py

```diff
@@ -34,6 +34,47 @@ class Classifier(Model):
                                                on_delete=models.CASCADE)
 
 
+class DistributionDigest(Model):
+    """
+    A model of digests on an individual distribution.
+    """
+    type = models.TextField()
+    digest = models.TextField()
+    project_specifier = models.ForeignKey("ProjectSpecifier",
+                                          related_name="digests",
+                                          related_query_name="distributiondigest",
+                                          on_delete=models.CASCADE)
+
+
+class ProjectSpecifier(Model):
+    """
+    A specifier of a python project.
+
+    Example:
+
+        version_specifier==1.0.0 will sync all distributions matching version
```

Comment:
I still think thes examples could be misleading. 

Also, s/sync/match/ throughout this docstring

Reply:
In what way are these examples misleading? (How can I clarify them?)

Reply:
`version_specifier==1.0.0` is not a valid version_specifier
I think even just adding punctuation will clarify 

"version_specifier: `==1.0.0` will match all distributions ..."